src/components/Editor/index.jsx

```diff
@@ -192,7 +198,7 @@ const Editor = (
           {isAttachmentsActive && (
             <Attachments
               attachments={attachments}
-              className="ne-attachments--integrated"
+              className={`ne-attachments--integrated ${attachmentsClassName}`}
```

Comment:
Use `classnames` package to use the second class if it is present. neetoEditor has similar usages you may refer to.